### PR TITLE
chore(dependabot): introduce

### PR DESCRIPTION
**Pull request by AugustinMauroy**
We need to be carful with this because It isn't documented.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2
+enable-beta-ecosystems: true
+updates:
+    - package-ecosystem: npm
```
Comment (anonrig) [inline .github/dependabot.yml:4]: We use pnpm, and unfortunately dependabot doesn't support pnpm.

Comment (AugustinMauroy) [reply .github/dependabot.yml:4]: See:
https://github.com/dependabot/dependabot-core/pull/7081
https://github.com/dependabot/dependabot-core/pull/7281

Comment (anonrig) [reply .github/dependabot.yml:4]: It seems it is only available through beta testing and the comments you shared mentions that using npm as the key does not update pnpm-lock file at the moment. I think we should wait for a release to be made before merging this.

Comment (AugustinMauroy) [reply .github/dependabot.yml:4]: Yeah good idea. So I close pr or put in draft ?

Comment (anonrig) [reply .github/dependabot.yml:4]: Let's keep it as draft. Thank you for your contribution.

Comment (AugustinMauroy) [reply .github/dependabot.yml:4]: You welcome 😁

Comment (AugustinMauroy) [reply .github/dependabot.yml:4]: https://github.blog/changelog/2023-06-12-dependabot-version-updates-now-supports-pnpm/

**Pull request merged**